### PR TITLE
#276 additional method to get an attachment

### PR DESCRIPTION
**Pull request by s22su**

### CODE REVIEW

**Comment by s22su** [comment]
@DavidGarciaCat yes. Will deal with that soon..

**Comment by Nyholm** [comment]
Thank you for this PR. 
The library is in the last stages to be rewritten. The RestClient is one of the classes that will be deprecated within the coming weeks. Just an FYI. The [new API's](https://github.com/mailgun/mailgun-php/milestone/1) should also support this feature. (That is nothing you have to do in this PR)

**Comment by s22su** [comment]
@Nyholm let's close this and #276 then and wait until better API is ready.

**Comment by Nyholm** [comment]
No, we can still merge this in. My personal idea is that version 3.0 is at least a year away. Just because a class is deprecated doesn't mean it will be removed anytime soon. 

I think it would be a good idea to support this feature for people using the "old way".

**Comment by s22su** [comment]
@Nyholm @DavidGarciaCat added this method also as a public method to Mailgun class and created test cases

File: tests/MailgunTest.php [hunk 1]
```diff
@@ -49,4 +49,22 @@ public function testVerifyWebhookEmptyRequest()
...
+
+        $this->assertInstanceOf('stdClass', $response);
+        $this->assertEquals($response->http_response_code, 200);
+    }
+
+    public function testGetAttachmentFail()
+    {
+        $this->setExpectedException('\\Mailgun\\Connection\\Exceptions\\GenericHTTPError');
+        $attachmentUrl = 'https://api.mailgun.net/non.existing.uri/1/2/3';
+        $client = new Mailgun('key-3ax6xnjp29jd6fds4gc373sgvjxteol0');
```
Comment (DavidGarciaCat) [inline tests/MailgunTest.php]: Is this your key? 😕 TBH I am not sure if is wise to provide the key as plain text here...

Comment (s22su) [inline tests/MailgunTest.php]: @DavidGarciaCat of course it is not my key. If you look a few lines up in the file then you can see the same key (line 26: https://github.com/mailgun/mailgun-php/blob/master/tests/MailgunTest.php#L26).

Also mailgun documentation uses this key in examples.

Comment (DavidGarciaCat) [inline tests/MailgunTest.php]: 👍 Thanks for clarify - I did the review out of home and had no time to double-check it.

**Comment by s22su** [comment]
Any chance to have it merged?

**Comment by Nyholm** [comment]
Yes, I'll merge this as a temporary fix. This feature will be replaced in the next major version.

**Pull request merged**